### PR TITLE
fix all code-checker error and warnings in Html2Text.php

### PR DESCRIPTION
**Pull request by naveenbalachandar**
Fixed the errors and warnings from code-checker in Html2Text.php.
With this PR all errors and warnings from code-checker have been fixed.
The accredible module plugin now follows the moodle coding style.

Screenshot 2021-10-04 at 2 27 09 PM

Screenshot 2021-10-04 at 2 26 00 PM

### CODE REVIEW

File: classes/Html2Text/Html2Text.php [hunk 1]
```diff
@@ -257,34 +257,32 @@ static function iterateOverNode($node, $prevName = null, $in_pre = false, $is_of
...
                 // Pass current node name to next child, as previousSibling does not appear to get populated.
-                if ($n instanceof \DOMDocumentType
-                    || $n instanceof \DOMProcessingInstruction
-                    || ($n instanceof \DOMText && static::isWhitespace($text))) {
-                    // Keep current previousSiblingName, these are invisible.
-                } else {
-                    $previousSiblingName = strtolower($n->nodeName);
+                if (!$n instanceof \DOMDocumentType
+                    && !$n instanceof \DOMProcessingInstruction
+                    && !($n instanceof \DOMText && static::is_whitespace($text))) {
```
Comment (naveenbalachandar) [inline classes/Html2Text/Html2Text.php:277]: Removed else from here as moodle coding style doesn't allow empty if block.

**Review by kei178 — CHANGES_REQUESTED** [review]
@naveenbalachandar - The current changes look good to me. I'd like to request additional two changes as follows:
 
1. Please add `defined('MOODLE_INTERNAL') || die();` at line#18 right after the `namespace` because it causes a warning below.

Screen Shot 2021-10-04 at 14 31 08

2. It does not raise any coding style errors or warnings, but it would be more appropriate to rename the directory and the class `Html2Text` to `html2text` to meet the requirement [here](https://docs.moodle.org/dev/Coding_style#Classes):

> Class names should always be lower-case English words, separated by underscores:

**Comment by kei178** [comment]
> 2. It does not raise any coding style errors or warnings, but it would be more appropriate to rename the directory and the class Html2Text to html2text to meet the requirement here:

@naveenbalachandar - Let’s leave it for now in case we want to override the Moodle core class `Html2Text` in the future.

**Review by kei178 — APPROVED** [review]
LGTM

**Pull request merged**